README.rdoc

```diff
@@ -46,7 +46,7 @@ contain just Markup-style markup (with or without leading '#' comment
 markers).  If directory names are passed to RDoc, they are scanned
 recursively for C and Ruby source files only.
 
-To generate documentation using +rake+ see RDoc::Task.
+To generate documentation using +rake+ see RDoc::Task[http://rake.rubyforge.org/classes/Rake/RDocTask.html].
```

Comment:
rubyforge is already sunsets. I should replace to http://docs.seattlerb.org/rdoc/RDocTask.html